src/audio/module_adapter/CMakeLists.txt

```diff
@@ -11,8 +11,13 @@ if((NOT CONFIG_LIBRARY) OR CONFIG_LIBRARY_STATIC)
 		module/volume/volume_hifi4.c
 		module/volume/volume_generic_with_peakvol.c
 		module/volume/volume_hifi3_with_peakvol.c
-		module/volume/volume_hifi4_with_peakvol.c
-		module/volume/volume.c)
+		module/volume/volume_hifi4_with_peakvol.c)
+
+	if(CONFIG_IPC_MAJOR_3)
+	add_local_sources(sof module/volume/volume_ipc3.c)
```

Comment:
indentation ?

Reply:
I checked following code, no indentation used, so I did not add indentation.

Reply:
ok, but the other code should use conditional indentation, so that can be fixed.